README.md

```diff
@@ -4,6 +4,58 @@ Loan and borrow!
 
 ## Quick links
 
-- [Front end](./be/)
+- [Front end](./fe/)
 
-- [Back end](./fe/)
+- [Back end](./be/)
+
+- [Database](./be/db/)
+
+## Table of contents
+
+- [Important variables](#important-variables)
+
+- [Running the development services](#running-the-development-services)
+
+- [Stopping the development services](#stopping-the-development-services)
+
+- [Troubleshooting](#troubleshooting)
```

Comment:
Please add these under `## Docker` or something similar, currently they look like unrelated paragraphs but I think they should be under a single heading.

Reply:
Fixed in commit 2f5fa9bd5ef486222acdb37cbfcda00f53a76282.